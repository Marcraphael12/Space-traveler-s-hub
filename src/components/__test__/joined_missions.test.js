import TestRenderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import JoinedMissions from '../missions/joined_missions';

describe('Render the missions list in profile page', () => {
  it('Renders all joined missions correctly into the profile page', () => {
    const missions = TestRenderer.create(
      <Provider store={store}>
        <JoinedMissions />
      </Provider>,
    ).toJSON();
    expect(missions).toMatchSnapshot();
  });
});