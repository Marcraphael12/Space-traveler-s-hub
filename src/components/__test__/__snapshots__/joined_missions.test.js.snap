// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the missions list in profile page Renders  1`] = `
<div
  className="missions-list"
>
  <h2>
    Joined missions
  </h2>
  <ul
    className="missions-list__missions"
  >
    <li
      className="missions-list__mission text-disabled"
    >
      No mission reserved...
    </li>
  </ul>
</div>
`;

exports[`Render the missions list in profile page Renders all joined missions 1`] = `
<div
  className="missions-list"
>
  <h2>
    Joined missions
  </h2>
  <ul
    className="missions-list__missions"
  >
    <li
      className="missions-list__mission text-disabled"
    >
      No mission reserved...
    </li>
  </ul>
</div>
`;

exports[`Render the missions list in profile page Renders all joined missions correctly into the missi 1`] = `
<div
  className="missions-list"
>
  <h2>
    Joined missions
  </h2>
  <ul
    className="missions-list__missions"
  >
    <li
      className="missions-list__mission text-disabled"
    >
      No mission reserved...
    </li>
  </ul>
</div>
`;

exports[`Render the missions list in profile page Renders all joined missions correctly into the profile page 1`] = `
<div
  className="missions-list"
>
  <h2>
    Joined missions
  </h2>
  <ul
    className="missions-list__missions"
  >
    <li
      className="missions-list__mission text-disabled"
    >
      No mission reserved...
    </li>
  </ul>
</div>
`;

exports[`Render the missions list in profile page Renders the missions list in profile page correctly 1`] = `
<div
  className="missions-list"
>
  <h2>
    Joined missions
  </h2>
  <ul
    className="missions-list__missions"
  >
    <li
      className="missions-list__mission text-disabled"
    >
      No mission reserved...
    </li>
  </ul>
</div>
`;
